mlir-sycl/test/Conversion/SYCLToLLVM/sycl-types-to-llvm.mlir

```diff
@@ -125,6 +126,10 @@ func.func @test_group(%arg0: !sycl_group_1_) {
 func.func @test_nd_item(%arg0: !sycl_nd_item_1_) {
   return
 }
+// CHECK: llvm.func @test_h_item(%arg0: !llvm.[[H_ITEM:struct<"class.sycl::_V1::h_item", \(]][[ITEM_1_FALSE]][[ITEM_BASE_1_FALSE]][[RANGE_1]][[ARRAY_1]][[SUFFIX]], [[ID_1]][[ARRAY_1]][[SUFFIX]][[SUFFIX]][[SUFFIX]], [[ITEM_1_FALSE]][[ITEM_BASE_1_FALSE]][[RANGE_1]][[ARRAY_1]][[SUFFIX]], [[ID_1]][[ARRAY_1]][[SUFFIX]][[SUFFIX]][[SUFFIX]], [[ITEM_1_FALSE]][[ITEM_BASE_1_FALSE]][[RANGE_1]][[ARRAY_1]][[SUFFIX]], [[ID_1]][[ARRAY_1]][[SUFFIX]][[SUFFIX]][[SUFFIX]][[SUFFIX]]) {
+func.func @test_h_item(%arg0: !sycl_h_item_1_) {
```

Comment:
move into alphabetical order (can you sort the file too pls).

Reply:
it is not completely sorted in alphabetical order, but I tried to sort it.